### ISSUE TITLE
Installing vol2birdR failure (proj not found)

### ISSUE THREAD

**Issue by peterdesmet** [adokter/vol2birdR#45]
Following the README, I did:

1. `brew install hdf5 proj gsl`
2. `devtools::install_github("adokter/vol2birdR")`

This resulted in this error:

```
configure: error: Proj not found in standard search locations. Install proj.4/proj.6 library
ERROR: configuration failed for package ‘vol2birdR’
* removing ‘/Library/Frameworks/R.framework/Versions/4.2-arm64/Resources/library/vol2birdR’
Warning message:
In i.p(...) :
  installation of package ‘/var/folders/dl/xs4s859d7077r33qyzfd6bx80000gn/T//RtmpTIcbKz/file14ab2460bd3bf/vol2birdR_0.2.2.9043.tar.gz’ had non-zero exit status
```

**Comment by PietrH** [comment]
I ran `sudo apt-get install proj-bin` when installing on Ubuntu

**Comment by peterdesmet** [comment]
Issue was resolved with `brew install pkg-config`

**Comment by adokter** [comment]
see also #27

**Comment by adokter** [comment]
fixed by #46
